content/03.results.md

```diff
@@ -76,36 +75,21 @@ A UMAP plot with cells colored by the total number of genes detected and a facet
-During conversion to `AnnData` objects, the modalities are exported as separate RNA (`_rna.h5ad`) and ADT (`_adt.h5ad`) objects.
+No filtering based on ADT expression or quality is performed; instead, we report QC statistics that users can employ for filtering before performing downstream analyses.
+The workflow also performs ADT-by-cell counts matrix normalization (see Methods for details).
 
 If a library contains ADT data, the QC report will include an additional section with a summary of ADT-related statistics, such as how many cells express each ADT, and ADT-specific diagnostic plots (Figure {@fig:figS2}B-D).
-As mentioned above, `scpca-nf` uses `DropletUtils::cleanTagCounts()` to calculate QC statistics for each cell using ADT expression but does not filter any cells from the object.
 We include plots summarizing the potential effects of removing low-quality cells based on RNA and ADT counts in the QC report (Figure {@fig:figS2}B).
-The first quadrant indicates which cells would be kept if the object was filtered using both RNA and ADT quality measures.
-The other facets highlight which cells would be removed if filtering was done using only RNA counts, only ADT counts, or both.
+The first quadrant indicates which cells would be kept if the object were filtered using both RNA and ADT quality measures.
+The other facets highlight which cells would be removed if filtering were done using only RNA counts, only ADT counts, or both.
 The top four ADTs with the most variable expression are also identified and visualized using density plots to show the normalized ADT expression across all cells (Figure {@fig:figS2}C) and UMAPs – calculated from RNA expression data – with cells colored by ADT expression (Figure {@fig:figS2}D).
 
 ### Multiplexed libraries
 
 To process multiplexed libraries, the HTO FASTQ files are quantified using `salmon alevin` and `alevin-fry` (Figure {@fig:figS2}C).
-Along with the FASTQ files, `scpca-nf` requires two TSV files to process multiplexed data: one to build an HTO-specific index for quantifying HTO expression with `alevin-fry`, and a second to indicate which HTO was used for which sample when multiplexing the library.
-The unfiltered HTO-by-cell counts matrix output from `alevin-fry` is saved as an `altExp` within the main `SingleCellExperiment` containing the unfiltered RNA counts.
-
-As with ADT data, `scpca-nf` does not filter any cells based on HTO expression, and any cells removed after filtering empty droplets based on the unfiltered RNA counts matrix are also removed from the HTO counts matrix in the filtered object.
-`scpca-nf` does not perform any additional filtering or processing of the HTO-by-cell counts matrix, so the same filtered matrix is included in the processed object.
+The unfiltered HTO-by-cell counts matrix output from `alevin-fry` is saved as an `altExp` within the main `SingleCellExperiment`, containing the unfiltered RNA counts.
```

Comment:
the comma here seemed odd, but I don't like my removal that much either.
```suggestion
The unfiltered HTO-by-cell counts matrix output from `alevin-fry` is saved as an `altExp` within the main `SingleCellExperiment` that contains the unfiltered RNA counts.
```

or maybe: 

```suggestion
As with ADT data, the HTO-by-cell counts matrix produced by `alevin-fry` is saved as an `altExp` within the main `SingleCellExperiment` object.
```